workspaces/simon-game/src/app/util/playSound.ts

```diff
@@ -0,0 +1,51 @@
+let audioContext: AudioContext | null = null;
+
+/**
+ * Plays a sound using the Web Audio API.
+ *
+ * @param context - The audio context to use for generating the sound.
+ * @param frequency - The frequency of the sound in Hertz
+ * @param durationMs - The duration of the sound in milliseconds.
+ * @param volumePct - The volume of the sound, typically between 0 and 1.
+ */
+export const playSound = (
+  frequency: number,
+  durationMs: number,
+  volumePct: number,
+) => {
```

Comment:
We've been generally explicitly annotating the return types of functions.

```suggestion
): void {
```